src/arrays.js

```diff
@@ -1,61 +1,71 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index > array.length - 1) {
+    return (array[index - array.length]);
+  } else {
+  return array[index];
+  }
```

Comment:
This could be done much more simply with the remainder operator:
```suggestion
  return array[index % array.length];
```